winui/DataGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,653 @@
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private static void DetailsViewExportHandler(object sender, DataGridDetailsViewExcelExportOptions e)
+{
+    e.ExcludedColumns.Add("OrderID");
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-export-without-column.png" alt="Excluding Columns in DetailsViewDataGrid while Exporting to Excel in WinUI DataGrid" width="100%" Height="Auto"/>
+
+
+Here, `OrderID` column is displayed in `DetailsViewDataGrid` and it is excluded while exporting to excel.
+
+### Customizing DetailsViewDataGrid cells
+
+Like parent DataGrid, You can customize the `DetailsViewDataGrid` cells also by using `CellsExportHandler`. Based on `DataGridCellExcelExportOptions.GridViewDefinition` property, you can identify the particular `DetailsViewDataGrid` and customize it.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.CellsExportHandler = CellsExportHandler;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private static void CellsExportHandler(object sender, DataGridCellExcelExportOptions e)
+{
+    if (e.GridViewDefinition == null)
+        return;
+
+    if (e.ColumnName == "OrderID")
+    {
+        e.Range.CellStyle.Font.Size = 12;
+        e.Range.CellStyle.Font.Color = ExcelKnownColors.Blue;
+        e.Range.CellStyle.Font.FontName = "Segoe UI";
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-exported-cell-customization.png" alt="Customizing DetailsViewDataGrid Cells while Exporting to Excel in WinUI DataGrid" width="100%" Height="Auto"/>
+
+## Customize exported workbook and worksheet
+
+SfDataGrid exports to excel by using [XlsIO](http://help.syncfusion.com/file-formats/xlsio/overview). You can refer [XlsIO documentation](http://help.syncfusion.com/file-formats/xlsio/working-with-excel-worksheet) for manipulating workbook and sheet after exporting. 
+
+### Workbook
+SfDataGrid provides option to return [ExcelEngine](https://help.syncfusion.com/cr/file-formats/Syncfusion.XlsIO.ExcelEngine.html) from that you can get exported workbook. This allows you to protect, encrypt and add worksheet before saving. 
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Worksheet customization
+
+SfDataGrid provides support to export to already existing file or worksheet. 
```

Comment:
The SfDataGrid provides support to export to an already existing file or worksheet.

Reply:
Content modified